src/team/teamView.js

```diff
@@ -3,9 +3,7 @@ import teamMemberView from './memberView'
 
 // A React element that displays all team members
 
-export default (teamMembersObj)  => 
-  (
-    <div>
-    {Object.keys(teamMembersObj).map(member => (teamMemberView(member)))}
-    </div>
-  )
\ No newline at end of file
+export default (teamMembersObj, id, styles)  => 
+  <div id={id}>
```

Comment:
Again, is there any way to move this element into the page template?  Or perhaps we don't need to target it by id?